core/templates/pages/exploration-editor-page/translation-tab/state-translation-editor/state-translation-editor.component.ts

```diff
@@ -22,6 +22,7 @@ require(
   'components/common-layout-directives/common-elements/' +
   'confirm-or-cancel-modal.controller.ts');
 require('services/external-save.service.ts');
+require('filters/format-base-translatable-object-values.filter.ts');
```

Comment:
I see a require for `format-base-translatable-object-values` but no usage of this filter in the ts file. Do we need to require the filter even if we are just using it in the template?